spec/kinde_sdk_spec.rb

```diff
@@ -230,7 +230,16 @@
 
     describe "api instances" do
       before do
-        stub_request(:get, "#{domain}/oauth2/user_profile")
+        stub_request(:get, "https://app.kinde.com/oauth2/user_profile").
```

Comment:
@viv-kinde We'll have to find a different stub, since that's querying prod. I'm not sure what this test is actually checking though, so I would probably remove it if we cannot stub it properly

Reply:
thanks @rairaman - removed test, seemed like it was making calls to app.kinde.com/oauth2.. even when i used example.com/oauth2..